### ISSUE TITLE
long strings push game buttons off screen when best time shows

### ISSUE THREAD

**Issue by maggiewiseman** [phetsims/graphing-lines#81]
Test device: Macbook Air

Operating System: OSX 10.12.4

Browser: Chrome: Version 58.0.3029.81 (64-bit)

Problem description:
Long strings that replace best time push the game buttons off the screen.

Steps to reproduce:
1.  Load the sim with ?stringTest=long query parameter
2. Choose games screen
3. Turn timer on
4. Win level 1 and level 2 so that best time shows up under game button on games screen. Level 3

Screenshots:
![12345678901234567890123456789012345678901234567890 screenshot 4](https://cloud.githubusercontent.com/assets/17385784/26286463/1ed42168-3e34-11e7-916d-2b79e2be014f.png)

![12345678901234567890123456789012345678901234567890 screenshot 5](https://cloud.githubusercontent.com/assets/17385784/26286461/183e9464-3e34-11e7-936f-9e5bc5788408.png)

Related to: phetsims/tasks/issues/843 
Troubleshooting information (do not edit):
Name: 12345678901234567890123456789012345678901234567890
URL: http://www.colorado.edu/physics/phet/dev/html/graphing-lines/1.2.0-rc.1/graphing-lines_en.html?stringTest=long
Version: 1.2.0-rc.1 2017-05-16 18:40:19 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.81 Safari/537.36
Language: en-US
Window: 1665x906
Pixel Ratio: 1.100000023841858/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"5c3f9204","branch":"HEAD"},"axon":{"sha":"f475e68c","branch":"HEAD"},"babel":{"sha":"4392e442","branch":"master"},"brand":{"sha":"f4b72cac","branch":"HEAD"},"chipper":{"sha":"3061f107","branch":"HEAD"},"dot":{"sha":"57936fff","branch":"HEAD"},"graphing-lines":{"sha":"dfc4ea9e","branch":"HEAD"},"joist":{"sha":"06775098","branch":"HEAD"},"kite":{"sha":"0cc5b620","branch":"HEAD"},"phet-core":{"sha":"76ec0204","branch":"HEAD"},"phetcommon":{"sha":"d0ffe176","branch":"HEAD"},"query-string-machine":{"sha":"e7055778","branch":"HEAD"},"scenery":{"sha":"d1d2ca2d","branch":"HEAD"},"scenery-phet":{"sha":"6dc1a064","branch":"HEAD"},"sherpa":{"sha":"0e326d54","branch":"HEAD"},"sun":{"sha":"3967c777","branch":"HEAD"},"tandem":{"sha":"30603e66","branch":"HEAD"},"vegas":{"sha":"6bd734e2","branch":"HEAD"},"vibe":{"sha":"5e8151e1","branch":"HEAD"}}

**Comment by pixelzoom** [comment]
This issue will never occur in practice, since a time is always formatted as (e.g.) 2:34:56.   But I'll look into addressing it so that it doesn't keep getting reported.

**Comment by pixelzoom** [comment]
I investigated, and this is actually a problem in vegas.LevelSelectionButton, where there is no `maxWidth` set for the time display:

```js
126 var bestTimeNode = new Text( '', { font: options.bestTimeFont, fill: options.bestTimeFill } );
```

Since it's in common code, I'm certainly not going to fix it for Graphing Lines.  And I don't think it's worth even fixing in common code since (as I said) it will never occur in practice.  

So.... Closing this issue as "won't fix".
